app/views/user_mailer/in_person_verified.html.erb

```diff
@@ -0,0 +1,60 @@
+<%= image_tag(
+      'email/user-signup-ial2.png',
+      width: 140,
+      height: 177,
+      alt: '',
+      class: 'float-center padding-bottom-4',
+    ) %>
+<h4><%= message.subject %></h4>
+<p>
+<div class="s16">
+  <%= t('user_mailer.in_person_verified.greeting') %><br>
+  <%= t(
+        'user_mailer.in_person_verified.intro',
+        location: @presenter.location_name,
+        date: @presenter.formatted_verified_date,
+      ) %><br /><br />
+  <%= t('user_mailer.in_person_verified.next_sign_in') %>
+</div>
+</p>
+
+<div class="mt-40">
+<table class="button expanded large radius">
```

Comment:
the padding and radius in the large button are a bit off from what is in the design

Reply:
Yeah, I figured it was better to match the button in every other email we send than define a custom appearance for the button in this one email, but I'll defer to designers' decision.

Reply:
That works for me! In the future, we need to revisit the email template in general, so will account for this in that work.